gui-tests/run_the_tests.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
```

Comment:
Please add also short description, what it does.

Reply:
There is no need for adding a description after all, because the script has been deleted (it was just a temporary script until the gradle issue is resolved).